modules/widgets/top-posts.php

```diff
@@ -455,14 +458,6 @@ function get_by_views( $count, $args ) {
 		 */
 		$days = (int) apply_filters( 'jetpack_top_posts_days', 2, $args );
 
-		if ( $days < 1 ) {
```

Comment:
Sorry for the delay in responding. The last commit I added deals with that scenario.